pkg/liqonet/iptables/iptables.go

```diff
@@ -653,9 +924,20 @@ func (h IPTHandler) insertLiqoRuleIfNotExists(chain string, rule IPTableRule) er
 }
 
 // Function to update specific rules in a given chain.
-func (h IPTHandler) updateSpecificRulesPerChain(chain string, existingRules []string, newRules []IPTableRule) error {
+func (h IPTHandler) updateSpecificRulesPerChain(
```

Comment:
Can we split this function in multiple sub-functions? I see two nested loops, and this is quite hard to read

Reply:
Do you think it could be useful? The inner loop it's only to check if an existing rule (from the outer loop) matches with one of the new rules: it's enough for a new function?